catalog/task/apply-mapping/README.md

```diff
@@ -0,0 +1,38 @@
+# apply-mapping
```

Comment:
I'm not sure I like having duplicate READMEs in both the version directory and the top level directory. I really like how you have the pipeline directory setup, so I suppose I am in favor of READMEs in the version directories but not top level directories. Maybe if we still want READMEs in both spots the top level one would be far less detailed... but that may be a little awkward to construct at least for the pipelines

Reply:
Thanks! 
I'm the top-level README is "optional", where the versioned README is "recommended". So I removed the top-level READMEs.